mySQL-example/MySQL.go

```diff
@@ -0,0 +1,183 @@
+package scripts
+
+import (
+	"database/sql"
+	"fmt"
+	"time"
+	//MySQL driver
+	_ "github.com/go-sql-driver/mysql"
+	//convert to strings
```

Comment:
What do you mean by saying "convert to string"? How is this related to the import over there?